src/test/resources/perimeters/cypress.json

```diff
@@ -1,18 +1,22 @@
 {
-	"id": "cypress",
-	"process": "cypress",
-	"stateRights": [
-		{
-			"state": "messageState",
-			"right": "ReceiveAndWrite"
-		},
-		{
-			"state": "messageStateWithNoAck",
-			"right": "ReceiveAndWrite"
-		},
-		{
-			"state": "messageStateWithOnlyWhenResponseDisabledForUserAck",
-			"right": "ReceiveAndWrite"
-		}
-	]
+  "id": "cypress",
```

Comment:
I had the following message in the console:

> WARNING : state responseState is present in perimeter for process cypress but not in process definition

I'm guessing responseState should be removed from this file or added in the cypress bundle?

Reply:
Removed from this file.